src/modinfo.h

```diff
@@ -43,6 +43,12 @@ class QDateTime;
 namespace MOBase { class IPluginGame; }
 namespace MOShared { class DirectoryEntry; }
 
+struct ModNameCompare {
```

Comment:
You can use the name comparator from `uibase` (in `ifiletree.h`). I think there is also one somewhere in this project but I don't know where.